src/k8s/pkg/utils/netlink.go

```diff
@@ -0,0 +1,59 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"os/exec"
+)
+
+type Vxlan struct {
```

Comment:
I think this is very generic, we can use something like:
```suggestion
type VXLANInterface struct {
```

Reply:
Done